gazebo_ros/include/gazebo_ros/gazebo_ros_api_plugin.h

```diff
@@ -139,14 +143,19 @@ class GazeboRosApiPlugin : public SystemPlugin
   void onModelStatesDisconnect();
 
   /// \brief Function for inserting a URDF into Gazebo from ROS Service Call
-  bool spawnURDFModel(gazebo_msgs::SpawnModel::Request &req,gazebo_msgs::SpawnModel::Response &res);
+  bool spawnURDFModel(gazebo_msgs::SpawnModel::Request &req,
+                      gazebo_msgs::SpawnModel::Response &res);
 
   /// \brief Both SDFs and converted URDFs get sent to this function for further manipulation from a ROS Service call
-  bool spawnSDFModel(gazebo_msgs::SpawnModel::Request &req,gazebo_msgs::SpawnModel::Response &res);
+  bool spawnSDFModel(gazebo_msgs::SpawnModel::Request &req,
+                     gazebo_msgs::SpawnModel::Response &res);
 
   /// \brief delete model given name
   bool deleteModel(gazebo_msgs::DeleteModel::Request &req,gazebo_msgs::DeleteModel::Response &res);
 
+  /// \brief delete a given light by name
+  bool deleteLight(gazebo_msgs::DeleteLight::Request &req,gazebo_msgs::DeleteLight::Response &res);
```

Comment:
the [roscpp style guidelines](http://wiki.ros.org/CppStyleGuide#Function_.2BAC8_Methods) don't specify this, so I guess my comment is ungrounded. but it does bother me that we are inconsistent throughout this repo on the style. as a moveit maintainer, i have setup clangformat to ensure style is automated and consistent, perhaps we should add that here (its not that hard) @j-rivero ?